rel-eng/origin-node.service

```diff
@@ -1,21 +1,19 @@
 [Unit]
-Description=OpenShift Node
+Description=Origin Node
 After=docker.service
-After=openshift-master.service
-After=openshift-sdn-master.service
-After=openshift-sdn-node.service
+After=origin-master.service
 Wants=docker.service
 Documentation=https://github.com/openshift/origin
 
 [Service]
 Type=notify
-EnvironmentFile=/etc/sysconfig/openshift-node
+EnvironmentFile=/etc/sysconfig/origin-node
 Environment=GOTRACEBACK=crash
 ExecStart=/usr/bin/openshift start node --config=${CONFIG_FILE} $OPTIONS
```

Comment:
You're right. Sorry about that. The error that tripped me up was actually the missing sysconfig file.